index.js

```diff
@@ -1,197 +1,240 @@
-let ToDoObjectsArray = []
+class App {
 
-document.getElementById('mainSubmitButton').setAttribute('onclick', createNewTodo)
-document.getElementById('clearAllButton').setAttribute('onclick', clearAllTasks)
+    #todos
 
-class toDoObject{
-    constructor(input) {
-        this.id = this.#createID()
-        this.isDone = false
-        this.text = input
+    constructor() {
+        this.todos = new Todos(() => this.printItems())
+        document.getElementById('mainSubmitButton').setAttribute('onclick', 'myApp.createNewTodo()')
```

Comment:
remember that you're in JS wolrd. You can use reference to the function rather than string eg.
```
this.clearAllTasks
```